cms/templates/howitworks.html

```diff
@@ -161,10 +161,10 @@ <h2 class="sr">${_("Sign Up for {studio_name} Today!").format(studio_name=settin
 
     <ul class="list-actions">
       <li class="action-item">
-        <a href="${reverse('signup')}" class="action action-primary">${_("Sign Up & Start Making Your {platform_name} Course").format(platform_name=settings.PLATFORM_NAME)}</a>
+        <a href="${settings.LMS_ROOT_URL + '/register'}" class="action action-primary">${_("Sign Up & Start Making Your {platform_name} Course").format(platform_name=settings.PLATFORM_NAME)}</a>
```

Comment:
**Reviewer Note:** Register URL could be hard-coded here (as was done), or made into a setting, or made into a function like `register_url_for_lms`.

I don't like the setting because it must be derived, and that requires it to be set in lms settings even though it isn't needed.  The function would be ok, but I couldn't think of a good home for it given the various places it was used.

Same is true for what was `FRONTEND_LOGOUT_URL`.

Reply:
What about the option of making it a CMS setting?  This would then allow Open edX instances to override it more easily in their own Django setting overrides.

Reply:
Maybe I'll add back as a setting if it makes it simpler for override.  Unfortunately, to make it work appropriately, it needs to be added to LMS as well.  I had it working that way before, so I can just restore it.

Reply:
ok.

Reply:
Settings have been restored.